contrib/README.md

```diff
@@ -0,0 +1,23 @@
+
+# Falco Contrib
```

Comment:
Given the adoption process we are going for (see https://github.com/falcosecurity/contrib/issues/14) I think we don't need a subdirectory here.

In fact, this repo will contain all the `kind/sandbox` Falco projects.

Reply:
As described in the corpus of #14 , the `evolution` repo will contain several things (like `/official`, see the proposed directory structure), not `kind/sandbox` projects only.

Have I misunderstood?

Reply:
Ok, resolved by https://github.com/falcosecurity/contrib/issues/14#issuecomment-632079475

Reply:
I will fix this PR soon